### ISSUE TITLE
Result Set from Stored Procedure Returns Only First Result

### ISSUE THREAD

**Issue by alexdavis24** [blue-yonder/turbodbc#323]
Running the following code:

```
# some params defined
cnxn = connect(driver='{ODBC Driver 17 for SQL Server}', server=srv, database=db, uid=user, pwd=password,
                            turbodbc_options=options)
crsr = cnxn.cursor()
cmd = "EXEC someDB.dbo.TEST_PROC"
crsr.execute(cmd)
result = crsr.fetchallnumpy()
print(result)
```

where `someDB.dbo.TEST_PROC`
is defined as 
```
CREATE PROC TEST_PROC
AS
SELECT 1
SELECT 2
```

Rather than getting the final result `2`, or both results, only `1` is returned.

Came across this issue when inserting into a temp table in a proc, followed by a select statement, which executed fine using SSMS. When moving to Python/turbodbc, however, I received an error that there was no available result set. Tried fetchall, fetchmany, etc. without much success. Any advice on how to work around this or explicitly pull the last result set would be greatly appreciated.

I am running ODBC Driver 17 for SQL Server, turbodbc 4.1.2 installed through conda-forge, and python 3.9.4. I've been using a turbodbc interface for a couple years, so I am moderately familiar with the package, but this behavior had me stumped!

**Comment by alexdavis24** [comment]
Does returning the this violate the Principal of Least Surprise - running a query in SSMS such as 
```
select 1
create table #tmptbl(val int)
```

or flipped
```
create table #tmptbl(val int)
select 1
```
will return a result set of 1, ie, a result set is always prioritized over a row count or return message. Wouldn't we expect the same behavior to be mirrored in our ODBC connections?

**Comment by david-engelmann** [comment]
This should be closed out due to #369
